jupyter_server_proxy/handlers.py

```diff
@@ -172,6 +172,8 @@ def _build_proxy_request(self, host, port, proxied_path, body):
             context_path = self._get_context_path(port)
             headers['X-Forwarded-Context'] = context_path
             headers['X-ProxyContextPath'] = context_path
+            # to be compatible with flask/werkzeug wsgi applications
+            headers['X_FORWARDED_PREFIX'] = context_path
```

Comment:
Should this be 'X-Forwarded-Prefix'? That is the more common HTTP Header naming convention. https://werkzeug.palletsprojects.com/en/1.0.x/middleware/proxy_fix/ seems to say that, but not sure if I'm looking in the right part of the documentation.

Reply:
you are right from the reference, but W3C says [Headers are case-insensitive](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html). do you want me to fix it?
Please let me now, I'll do another PR.
NB: I have a working case under test..

Reply:
Ah, I was more worried about the '_' instead of '-', rather than the case.

Reply:
Ok, my mistake, I verify!

Reply:
Funy enough, it's what I have (and working well) on my test platform: I make the change and verify it works.

Reply:
Just tested the change and it works as well! 

Do I make a new pull request?